app/controllers/people_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    @task = Task.all
+
+    Task.where(:completed_at => nil).count
+
+    render :index
+
+  end
+
+  def show
+    @task = Task.find(params[:id])
+
+    @person_id = @task.person_id
+    @person = Person.find(params[:id])
```

Comment:
Your view for show is only using the `@person` instance variable. Therefore, you don't need the `@task` or `@person_id` instance variables, you can just use local variables for this that the view doesn't need to have access to.